src/media/css/feed/collections.styl

```diff
@@ -50,15 +54,107 @@
     }
 }
 
+.collection-landing-wrapper:not([data-collection-landing-is-listing]) .collection-landing-header-mobile {
```

Comment:
Comment? `data-collection-landing-is-listing` is a confusing name `[data-collection-landing~=listing]`?